co-log-polysemy/co-log-polysemy.cabal

```diff
@@ -0,0 +1,96 @@
+cabal-version:       2.4
+name:                co-log-polysemy
+version:             0.0.0.0
+synopsis:            Composable Contravariant Comonadic Logging Library
+description:
+    Implementation of the [co-log](http://hackage.haskell.org/package/co-log-core)
+    logging based on the [polysemy](http://hackage.haskell.org/package/polysemy)
+    extensible effects library.
+    .
+    The ideas behind this package are described in the following blog post:
+    .
+    * [co-log: Composable Contravariant Combinatorial Comonadic Configurable Convenient Logging](https://kowainik.github.io/posts/2018-09-25-co-log)
+    .
+    For the taggless final implementation of the @co-log@ ideas se the following package:
+    .
+    * [co-log](http://hackage.haskell.org/package/co-log)
+
+homepage:            https://github.com/kowainik/co-log
+bug-reports:         https://github.com/kowainik/co-log/issues
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Kowainik
+maintainer:          xrom.xkov@gmail.com
+copyright:           2019 Kowainik
+category:            Logging, Contravariant, Comonad, Effects
+build-type:          Simple
+stability:           provisional
+extra-doc-files:     CHANGELOG.md
+                   , README.md
+tested-with:         GHC == 8.2.2
```

Comment:
Really? I'm like 95% sure polysemy won't compile below 8.6 --- not for any fundamental reasons other than it's using some unnecessary `QuantifiedConstraints` right now.

Reply:
I wrote these lines before I tested on the CI and noticed that `polysemy` doesn't build on GHC < 8.6. That's why I have the following lines in the `.cabal` file:

```
  if impl(ghc < 8.6)
      buildable:       False
```

Just forgot to update metadata. Thanks for noticing this!

Reply:
https://github.com/isovector/polysemy/issues/29 is the ticket for this work on `polysemy` --- I'm traveling for a few weeks but can tackle it when I'm home. Should't be more than a ten minute change!